rules/aws/impact_rds_group_deletion.toml

```diff
@@ -0,0 +1,57 @@
+[metadata]
+creation_date = "2021/06/05"
+maturity = "production"
+updated_date = "2021/06/05"
+
+[rule]
+author = ["Elastic", "Austin Songer"]
+description = """
+Identifies when a RDS Security Group deleted.
+"""
+false_positives = [
+    """
+    A RDS group may be deleted by a system administrator. 
```

Comment:
```suggestion
        A RDS security group deletion may be done by a system or network administrator. Verify whether the user
    identity, user agent, and/or hostname should be making changes in your environment. Security Group deletions from unfamiliar
    users or hosts should be investigated. If known behavior is causing false positives, it can be exempted from the
    rule.
```
re-lint